mxnet-bot/PredictLabels/Trainer.py

```diff
@@ -21,29 +21,48 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.svm import SVC
 from sklearn.preprocessing import LabelEncoder
+import tempfile
 import pickle
 import logging
 
 
 class Trainer:
+    # target labels that we are interested in
+    labels = ["Performance", "Test", "Question",
+               "Feature request", "Call for contribution",
+               "Feature", "Example", "Doc",
+               "Installation", "Build", "Bug"]
 
-    def __init__(self):
+    def __init__(self, 
+                 tv=TfidfVectorizer(min_df=0.00009, ngram_range=(1, 3), max_features=10000), 
+                 clf=SVC(gamma=0.5, C=100, probability=True),
+                 tmp_tv_file=tempfile.NamedTemporaryFile(),
```

Comment:
Why don't we put all of them in a temporary directory? I think this would be cleaner. The file names can be hardcoded in this temporary directory.

Reply:
Done. Put all of them in a tempfile.TemporaryDirectory